dtc/PKGBUILD

```diff
@@ -0,0 +1,27 @@
+# Maintainer: Andy Gross <andy.gross@linaro.org>
+
+pkgname=dtc
+pkgver=1.4.4
```

Comment:
yeah i noticed that.  The odd thing is locally, it builds fine.  We need to stick with tagged versions.  I am modifying the patch to fix the FPIC stuff and will add it to the build once I get the right format on it.